client/gutenberg/extensions/map/map-theme-picker/style.scss

```diff
@@ -2,7 +2,7 @@
 
 .component__map-theme-picker {
 }
-.component__map-theme-picker__button {
+.component__map-theme-picker__button.components-button {
```

Comment:
I wonder why we need the increased specificity here . FWIW, the properties that are otherwise overridden (and lead to the visual glitch documented in the PR desc) are

```css
background-position: center center;
background-size: contain;
```

Reply:
(The weird thing is that this works in wp-admin without the extra specificity.)